tavern/tomes/get_net_info/main.eldritch

```diff
@@ -0,0 +1,15 @@
+def get_net_info():
+    print("Interfaces:\n\n")
+    for interface in sys.get_ip():
+        for key, value in interface.items():
+            if key == "ips":
+                for ip in value:
+                    print("ip: {}\n".format(ip))
+            else:
+                print("{}: {}\n".format(key, value))
+        print("\n")
+    print("Hostname:\n\n" + sys.hostname())
```

Comment:
One nit.
Can we put hostname in line with the header as you do with mac and ip.
Should just be moving the newlines to the end.
Eg. `Hostname: hultos-laptop`